gtsfm/utils/metrics.py

```diff
@@ -57,23 +57,23 @@ def count_correct_correspondences(
     return np.count_nonzero(epipolar_distances < epipolar_dist_threshold)
 
 
-def compute_errors_statistics(errors: List[Optional[float]]) -> StatsDict:
+def compute_errors_statistics(errors: List[Optional[float]], num_sig_figs: int = 2) -> StatsDict:
```

Comment:
Can we somehow make this a global param? I feel we might need to do this everywhere?

Reply:
I think we could put it at the top of the file, perhaps.

But I think more than 2 sig figs would not be useful for anyone, and I think less than 2 sig figs is also two few. What if I make 

```python
PRINT_NUM_SIG_FIGS = 2
```
at the top of the file?